content/post/mv-reflection-001.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "Max's first reflection"
+date = 2024-06-11T12:44:15-04:00
+authors = ["Max Veytsman"]
++++
+
+We're some number[^1] of weeks into this thing, and I owe myself and my fellow fellows a reflection of some kind.
+
+There's a sense that I had when I was trying to frame my research of topic where I felt "have I been asking myself this question my entire career?"
+
+This feels a little true for all the fellowships: Xavier's is explicitly self-ethnography. I think what Serena is looking at around community and care is deeply personal: in that her work making space at 1RG by her very presence is such a part of it. I don't have as strong of an understand of Cheryl's project yet but it *felt* very personal and rooted in the self when she described it at our last meeting.
+
+This did lead me to ask myself a silly question: "is all research, when you really come down to it, an act of self-ethnography?"
```

Comment:
This reminds me of that page I just sent you about [philosopical idealism](https://plato.stanford.edu/entries/idealism/) which defined epistemological idealism thus:


> although the existence of something independent of the mind is conceded, everything that we can know about this mind-independent “reality” is held to be so permeated by the creative, formative, or constructive activities of the mind (of some kind or other) that all claims to knowledge must be considered, in some sense, to be a form of self-knowledge.